tests/integration_tests/api_v2/model.go

```diff
@@ -234,16 +234,16 @@ type Table struct {
 // SinkConfig represents sink config for a changefeed
 // This is a duplicate of config.SinkConfig
 type SinkConfig struct {
-	Protocol                 string            `json:"protocol"`
-	SchemaRegistry           string            `json:"schema_registry"`
-	CSVConfig                *CSVConfig        `json:"csv"`
+	Protocol                 string            `json:"protocol,omitempty"`
```

Comment:
Whether all fields should use Pointer?

Reply:
I think we do not need to use pointer for a string, as an empty string `var s string = ""` will be ignored automatically.